life/modernized/02-life.adoc

```diff
@@ -1146,6 +1146,9 @@ I had not been here this day.`"
 [.offset]
 __Which Testimony was as follows:__
 
+[.embedded-content-document.letter]
+--
+//not sure about this one
```

Comment:
Yep, I think this is right. Since its a short, written testimony that she signed, I think we should embed it.  However, did Jared show you that there are other options for the embedded sections besides letters? This one I would actually label `[.embedded-content-document.testimony]`  Note how I changed the last word from _letter_ to _testimony_. If you look in the Style Guide, there are five possible tags for embedded documents.